packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.css

```
@@ -12,3 +12,17 @@ ion-card {
 geov-paginator {
   margin-left: auto;
 }
+
+.litteral-container {
```

```suggestion
.literal-container {
```
Literal just has a single t.

---

Done